### PR TITLE
Space vacuum damage doubled

### PR DESCRIPTION
**Pull request by nicbn**
[Changelogs]: #
:cl: nicn
balance: Damage from low pressure has been doubled.
/:cl:

Ok so the reasons

* Space is not supposed to be something to fuck with.
* Spacewalk without proper suits is not too much damaging and this is quite annoying to traitors dragging off people into space and people fighting back while in space and people getting out of airlocks to run for a traitor.
* WHY NOT
* Really, if we find out this to be bad for players we can just revert it

### CODE REVIEW

**Comment by SyruseEln** [comment]
Space is a joke on tg. this change is required.

**Comment by Qbopper** [comment]
if people think this is something that would be good to change then you'd probably want to have it testmerged and see if it ruins space or whatever

i like the idea of pressure being more dangerous though, might encourage people to fix breaches more often

**Comment by AnonymousNow** [comment]
I remember the days of space stabilisation. When you could walk into a room exposed to space, take some damage, and then your system would stabilise and you wouldn't take any more damage so long as you weren't on a space tile.

It was broken and unrealistic, but on occasion I miss those times.

**Review by ChangelingRain — APPROVED** [review]

**Comment by XDTM** [comment]
If you're changing fundamental space mechanics could you make it not cold and instead up the pressure damage to match

**Comment by nicbn** [comment]
@XDTM Yeah I'm planning to do this too in another PR

**Review by duncathan — APPROVED** [review]

**Comment by epochayur** [comment]
maybe someday we'll get proper explosive decompressions
good pr though

**Comment by duncathan** [comment]
$250 USD for the first 20 hours of work + $15 USD per hour for additional hours and I will make explosive decompression

**Comment by praisenarsie** [comment]
@optimumtact ^^^^^^^^^^^

**Comment by ACCount12** [comment]
Spacewalking is balanced as is: it's hard enough to pull off that it makes people freak out when you do that. 👎

**Comment by nicbn** [comment]
I'd rather testmerge (or merge) this and let the players decide.

**Comment by epochayur** [comment]
if a single person has ever been impressed by an antag jumping into space as an easy escape route, I have yet to meet them

**Comment by The-clown-known-as-Pepe** [comment]
>WHY NOT

👎

**Review by Cyberboss — APPROVED** [review]

**Comment by ACCount12** [comment]
This is the type of PR I hate the most: changing things for the sake of changing things.

**Comment by Cyberboss** [comment]
ITT: People arguing that humans can go into space unprotected and being mostly OK afterwards

**Comment by epochayur** [comment]
someone hasn't read hitchhiker's guide

**Comment by Cyberboss** [comment]
What actually should be happening:
https://youtu.be/3LEa0FN1bf8?t=98

**Comment by PKPenguin321** [comment]
>a realism argument

**Comment by Bawhoppen** [comment]
Realism does actually matter contrary to what people seem to think

**Comment by PKPenguin321** [comment]
..no

**Comment by epochayur** [comment]
I don't think it's realism, more "is this grounded for the world in which it's presented" and to that I say absolutely, space should be a cruel mistress in this universe

**Comment by AdamElTablawy** [comment]
realism argument isn't really important for the sake of realism, so much as realism can be important for the sake of not looking stupid

ie, you can say 'we have fucking wizards and magic why are we talking about realism', but then that's just a justification to have something that doesn't really make much sense mechanics wise or when you think about it vs having something unrealistic but entertaining

**Comment by KorPhaeron** [comment]
Needs to be test merged first

**Comment by AnonymousNow** [comment]
I think the notion of imbibing a chemical slurry and putting on a firesuit to get one more precious minute in space is pretty fundamental to the feeling of SS13.

**Comment by Bawhoppen** [comment]
We can't live in the past forever. SS13 has to change some day, and that day is today.

**Comment by AnonymousNow** [comment]
Is it progress, though?

It feels very much like change for the sake of change, which is a great way to make a project not feel like that project anymore. You can see a similar logic in most of Goof's PRs.

**Comment by duncathan** [comment]
Just so it's clear I'm very open to the idea of only increasing to 3, not 4. I still want you to be able to survive a spacewalk using ghetto heating - just want it to be harder so that doing so is even more meaningful. I want space to be deadly without destroying interesting things like that

**Comment by duncathan** [comment]
I don't know what the exact reasons for this particular PR are but it aligns with my goal of making the environment more deadly. In particular, I want a relevant and constant opposing force to lowpop crews in extended where there are too few players for antags. Since our environment is by nature quite hostile and I'm familiar with environmental things, making space more deadly is the obvious answer. 

tl;dr don't assume there isn't a reason

**Comment by ACCount12** [comment]
We already have the event system here to ruin the day for anyone who attempts extended.

**Comment by Bawhoppen** [comment]
@AnonymousNow Progress is not inevitable. Change, however, is.

**Comment by duncathan** [comment]
We do have the event system. I want more. You can disagree with me but don't pretend I don't have a reason.

**Comment by Steelpoint** [comment]
Space should be very dangerous for anyone with no protection to the elements. I would be interested in seeing this change in game.

**Comment by PKPenguin321** [comment]
That's stupid baw

**Comment by Bawhoppen** [comment]
I'm saying we should be willing to try new things, and not just do the same stuff that has always worked.

Also what I'm kind of talking about life in general more than spessmen.

**Comment by Iatots** [comment]
>progress happening without change
>progress being an inevitability

that's really fucking optimistic

**Comment by Bawhoppen** [comment]
I said Progress was NOT inevitable, but change, however, was.

**Comment by KorPhaeron** [comment]
Don't think this ever got test merged or will be any fun but I guess it is easy to revert when people realize that breaking a shuttle window and everyone keeling over in transit or setting off a bomb and people die too quickly to move through hallways is unfun

**Comment by duncathan** [comment]
Yeah I'm very open to toning this down or even reverting if it proves to be way too much

**Comment by PKPenguin321** [comment]
Saying that change must happen with time is literally the same argument behind saying "its not previous year anymore, it's the current year" btw

**Comment by Iatots** [comment]
Am I tripping or did you edit your comment?

**Comment by Bawhoppen** [comment]
I didn't edit it
